### PR TITLE
Add direct-from-marketing.yml issue template

### PR DESCRIPTION
**Pull request by KiraFuruichi**
Add direct-from-marketing template for marketing-approved pieces

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](../contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](../website/docs/docs/guides/migration-guide)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by KiraFuruichi** [comment]
Hey @gwenwindflower can I get your review on this plz 🙏

File: .github/ISSUE_TEMPLATE/direct-from-marketing.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      description: Please link to the artifact (content brief/event recap) you are using to guide your work.
+    validations:
+      required: false
+      
+  - type: textarea
+    attributes:
+      label: Link to the initial outline we will develop through this Issue.
+      description: Please link to the initial outline in FigJam, Notion, or Google Docs
+    validations:
+      required: true
```
Comment (gwenwindflower) [inline .github/ISSUE_TEMPLATE/direct-from-marketing.yml:38]: do you want this to be required? i'm just wondering if you'll always have a public facing outline necessarily for these

File: .github/ISSUE_TEMPLATE/direct-from-marketing.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      placeholder: your preferred email and/or dbt Slack handle
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: from-discussion
+    attributes:
+      label: This issue is being made to create a marketing-approved piece.
+      description: >
+        This content will typically fall under glossary pages, community recap events, and devblog content for campaigns. Please reach out to John Blust if there's any confusion here.
```
Comment (gwenwindflower) [inline .github/ISSUE_TEMPLATE/direct-from-marketing.yml]:
```suggestion
        This content will typically fall under glossary pages, community recap events, and devblog content for campaigns. Please reach out to @johnblust if there's any confusion here.
```

File: .github/ISSUE_TEMPLATE/direct-from-marketing.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      options:
+        - label: I am directly creating this issue for a marketing-approved piece.
```
Comment (gwenwindflower) [inline .github/ISSUE_TEMPLATE/direct-from-marketing.yml]: Do you perhaps want to make this even more limited? Like, 'I work for dbt Labs and am directly etc etc' -- I'm not sure if there would ever be a reason for somebody outside the company to create this type of issue, and if not, probably good to add for extra clarity.

File: .github/ISSUE_TEMPLATE/direct-from-marketing.yml (hunk 1 above)
Comment (gwenwindflower) [reply .github/ISSUE_TEMPLATE/direct-from-marketing.yml:38]: Maybe you even just merge this one into the one above it for this type?

**Review by gwenwindflower — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/direct-from-marketing.yml (hunk 3 above)
Comment (KiraFuruichi) [reply .github/ISSUE_TEMPLATE/direct-from-marketing.yml]: Good callout!

File: .github/ISSUE_TEMPLATE/direct-from-marketing.yml (hunk 2 above)
Comment (KiraFuruichi) [reply .github/ISSUE_TEMPLATE/direct-from-marketing.yml]: 💯

File: .github/ISSUE_TEMPLATE/direct-from-marketing.yml (hunk 1 above)
Comment (KiraFuruichi) [reply .github/ISSUE_TEMPLATE/direct-from-marketing.yml:38]: I think I'm just going to merge into one section. For the most part, our outlines here should be public facing and we've been putting our outlines in the Issues so far. Will change this in the future if we're feeling it's no longer appropriate!

Comment (KiraFuruichi) [reply .github/ISSUE_TEMPLATE/direct-from-marketing.yml:38]: Ok made some minor changes, thanks for your review!

**Review by gwenwindflower — APPROVED** [review]
perrrf :shipit:

**Pull request merged**